galley/pkg/config/analysis/diag/message_test.go

```diff
@@ -29,11 +31,10 @@ func TestMessage_String(t *testing.T) {
 	g.Expect(m.String()).To(Equal(`Error [IST-0042] Cheese type not found: "Feta"`))
 }
 
-func TestMessageWithOrigin_String(t *testing.T) {
+func TestMessageWithResource_String(t *testing.T) {
 	g := NewGomegaWithT(t)
-	o := testOrigin("toppings/cheese")
 	mt := NewMessageType(Error, "IST-0042", "Cheese type not found: %q")
-	m := NewMessage(mt, o, "Feta")
+	m := NewMessage(mt, &resource.Instance{Origin: testOrigin("toppings/cheese")}, "Feta")
```

Comment:
Could create this test resource just once and re-use it in tests.

Reply:
Spoke in person and I expressed a strong preference to leave these embedded as the "toppings/cheese" string is an important detail of the test. Leaving it here makes it (admittedly slightly) easier to verify correctness at a glance.